### DISCUSSION TITLE
隔壁 YiriMirai 前来观光

### DISCUSSION THREAD

**Discussion by Wybxc** [Excaive/miraicle#3] [General]
Graia 马上要发布 V5（Avilla）了，不过看起来使用体验并不是很良好……Avilla 试图兼容 Onebot 协议，我认为这并不是一个很好的方向。

现在 Graia Application 已经挂上了“随缘更新”的公告，再加上至今没有着落的文档，让人很难对 Graia 未来的生态抱有期望。

处于这一点，我开发了 YiriMirai，希望能打造一个易用性良好的社区 SDK。

在开发 YiriMirai 的过程中，我发现了 miraicle 这样一个优秀的项目。我相信，我们这样的项目也会收到更多人的青睐。

一起共勉吧。

**Comment by Excaive** [comment]
你好，欢迎来访。

我没有使用过 `Graia`，但用过它的前身 `python-mirai(kuriyama)`。当时我在阅读了它的文档之后，仍然难以掌握它的用法，所以产生了写一套 sdk 的想法。在这个过程中，也参考了包括 `python-mirai` 在内的一些项目。不过由于耦合度高的问题，一直处于自己使用的状态，直到最近才解决了问题并且开了源，后来在写文档的时候也参考了 `YiriMirai`。

`Graia` 现在发布 `Avilla`，原项目随缘更新了，这对 `mirai `的 `python` 生态可能会有影响，不过对于我们刚起步的项目来说也是一个机遇吧。也祝 `YiriMirai` 受到更多人的青睐，共勉。
